app/views/mortgage_calculator/stamp_duties/_form_step1.html.erb

```diff
@@ -5,7 +5,7 @@
     <%= f.label :price, class: "stamp-duty__input-description" %>
     <div class="visually-hidden" id="mc_accessibility_describe_price"><%= t 'stamp_duty.describe_price_field' %></div>
 
-    <span class="stamp-duty__input--wrapper">
+    <span class="stamp-duty__input--wrapper stamp-duty__input-wrapper--inline">
```

Comment:
Hey @JiggyPete another small one, here you have ``stamp-duty__input--wrapper`` and ``stamp-duty__input-wrapper--inline`` as the modifier.

In this case the first class ``stamp-duty__input--wrapper`` should be ``stamp-duty__input-wrapper``.  And the 2nd class is the modified version (``stamp-duty__input-wrapper--inline``)